### ISSUE TITLE
Redirect / pipe stdout does not work

### ISSUE THREAD

**Issue by hholst80** [stedolan/jq#1351]
Excuse me if this has been fixed in the latest version.

```
$ jq --version
jq-1.5-1-a5b5cbe
```

This works as expected:

```
$ < /tmp/data.json jq -M 
[
  {
    "name": "atari-py",
    "version": "0.0.18"
  },
  {
    "name": "box2d-py",
    "version": "2.3.1"
  },
  {
    "name": "conda",
    "version": "4.2.12"
  }
]
```

but redirecting the stdout does not work:

```
$ < /tmp/data.json jq -M > /tmp/data2.json
jq - commandline JSON processor [version 1.5-1-a5b5cbe]
Usage: jq [options] <jq filter> [file...]

        jq is a tool for processing JSON inputs, applying the
        given filter to its JSON text inputs and producing the
        filter's results as JSON on standard output.
        The simplest filter is ., which is the identity filter,
        copying jq's input to its output unmodified (except for
        formatting).
        For more advanced filters see the jq(1) manpage ("man jq")
        and/or https://stedolan.github.io/jq

        Some of the options include:
         -c             compact instead of pretty-printed output;
         -n             use `null` as the single input value;
         -e             set the exit status code based on the output;
         -s             read (slurp) all inputs into an array; apply filter to it;
         -r             output raw strings, not JSON texts;
         -R             read raw strings, not JSON texts;
         -C             colorize JSON;
         -M             monochrome (don't colorize JSON);
         -S             sort keys of objects on output;
         --tab  use tabs for indentation;
         --arg a v      set variable $a to value <v>;
         --argjson a v  set variable $a to JSON value <v>;
         --slurpfile a f        set variable $a to an array of JSON texts read from <f>;
        See the manpage for more options.
```

**Comment by pkoppstein** [comment]
This problem has been addressed -- in 5fe053671a768d28bc9773719cc4a9a35e72f30d of Dec 2 2015 I believe (after the official release of jq 1.5).

**Comment by hholst80** [comment]
Thank you!
